src/main/java/com/danielflower/apprunner/runners/GoRunner.java

```diff
@@ -0,0 +1,70 @@
+package com.danielflower.apprunner.runners;
+
+import com.danielflower.apprunner.problems.ProjectCannotStartException;
+import org.apache.commons.exec.CommandLine;
+import org.apache.commons.exec.ExecuteWatchdog;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.maven.shared.invoker.InvocationOutputHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+public class GoRunner implements AppRunner {
+    public static final Logger log = LoggerFactory.getLogger(GoRunner.class);
+
+    public static final String[] startCommands = new String[]{"go build", "go test", "./{app_dir_name}"};
```

Comment:
These should be used by the code to force them to be up to date. It's ugly - `startCommands[0]` - but at least the documentation will be kept up to date.